stellar-auth/pkg/cli/add_user.go

```diff
@@ -40,7 +40,7 @@ func AddUserCmd(databaseURLFlagName string, passwordPrompt PasswordPromptInterfa
 		},
 		{
 			Name:        "password",
-			Usage:       "Sets the user password, it should be at least 8 characters long, if omitted, the command will generate a random one.",
+			Usage:       "Sets the user password, it should be at least 12 characters long, if omitted, the command will generate a random one.",
```

Comment:
Can't you make minPasswordLength and maxPasswordLength into public values and use them here, rather than hardcoding the number `12` again?